keps/sig-storage/0033-20181111-transfer-pvc-between-namespaces.md

```diff
@@ -0,0 +1,179 @@
+---
+kep-number: 33
+title: Add ability to transfer PVC between Namespaces
+authors:
+  - "@j-griffith"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-architecture
+reviewers:
+  - TBD
+approvers:
+  - TBD
+editor: @j-griffith
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+status: provisional
+---
+
+# Kubernetes PVC Namespace Transfer
+
+
+## Table of Contents
+
+A table of contents is helpful for quickly jumping to sections of a KEP and for highlighting any additional information provided beyond the standard KEP template.
+[Tools for generating][] a table of contents from markdown are available.
```

Comment:
This generic blurb from the template can be removed.

Reply:
Oops, thanks